torch/csrc/jit/tracer.h

```diff
@@ -74,6 +74,23 @@ inline Value* getValueTrace(const Variable& var) {
   return it->second;
 }
 
+inline Value* getNestedValueTrace(const std::shared_ptr<TracingState>& state, const IValue &v) {
```

Comment:
Can we just make this an overload of `getValueTrace` that works on IValue instead of Tensor?

Reply:
So when I do this, I run into trouble for ambiguous functions calls.
I then tried to add a dummy template parameter to de-prioritize the `IValue` overload. This works for "regular calls", but fails when `getValueTrace` is used in fmap driectly as `fmap(value, getValueTrace)`.

I would need a hint how to do this cleanly.

The obvious alternative would be to only offer the `IValue` version and move the body of the `Variable` version to the bottom of that. However, I'm not sure whether that has undesirable implications in the callers (performance or otherwise).

I've tentatively changed back to using two distinct names, but with your advice, will improve it.

Reply:
I see. Two distinct names would be better than more complicated alternatives.

Reply:
Then I'd claim to be done. :)